### PR TITLE
Add the z=0 EAGLE-100 Ref stellar mass - halo mass relation

### PR DESCRIPTION
**Pull request by MatthieuSchaller**
What do you think of the format? 

If that works, I'll add the 25Mpc box and other redshifts.

### CODE REVIEW

**Comment by JBorrow** [comment]
Looks good, but can we change `.dat` for `.txt` or `.tsv`? `.dat` generally are binary files

**Comment by MatthieuSchaller** [comment]
Also, can I use the same script to also produce M_200 - M_*/M_200 or do you want one separate script per dataset?

**Comment by JBorrow** [comment]
Either is fine

**Comment by JBorrow** [comment]
do you want to do that here or in a separate mr

**Comment by MatthieuSchaller** [comment]
Working on it. Give me 1hr.

**Comment by JBorrow** [comment]
no worries any time is fine just wanted to know whether to merge or not

**Comment by MatthieuSchaller** [comment]
Done. :)

**Comment by JBorrow** [comment]
So this is definitely a fit not a median line?

**Comment by JBorrow** [comment]
Can you also change the comment to indicate how the relation has been reduced in those cases?

**Comment by MatthieuSchaller** [comment]
It's the median line.

**Comment by JBorrow** [comment]
Ok, we should change the description to state that instead of fit then

**Comment by MatthieuSchaller** [comment]
Done.

**Comment by JBorrow** [comment]
Sorry can you also change the name to indicate that they are the reduced versions in those two scripts

**Comment by MatthieuSchaller** [comment]
What do you mean?

**Comment by MatthieuSchaller** [comment]
The dedscription already says medians.

File: data/GalaxyStellarMassHaloMass/conversion/convertEAGLERatioStellar.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+# Meta-data
+comment = (
+    "Medians obtained directly from the subfind catalogs. "
+    "Central galaxies only. Halos with 0 stellar mass included. "
+    "No cosmology correction needed."
+)
+citation = "EAGLE - L100N1504"
+bibcode = "2015MNRAS.446..521S"
+name = "Stellar mass - halo mass relation at z=0."
```
Comment (JBorrow) [inline data/GalaxyStellarMassHaloMass/conversion/convertEAGLERatioStellar.py:41]: Change this to "Galaxy Stellar Mass / Halo Mass - Stellar Mass relation at z=0" for instance.

**Comment by MatthieuSchaller** [comment]
Done.

**Pull request merged**